app/style/components/participant-avatar.less

```diff
@@ -139,13 +139,6 @@ participant-avatar {
     .avatar-size(xxs);
   }
 
-  &.avatar-l,
-  &.avatar-xl {
-    .avatar-border {
-      display: none;
-    }
-  }
```

Comment:
Is this change also valid for users?

Reply:
The border is barely visible as it is. I'd rather be consistent.